components/well-bold/well-bold-fr.html

```diff
@@ -0,0 +1,75 @@
+---
+breadcrumb: [
+	{ "title": "GCWeb home", "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-fr.html" },
+	{ "title": "Fonctionalités provisoires", "link": "https://wet-boew.github.io/themes-dist/GCWeb/provisional-fr.html" }
+]
+dateModified: "2021-05-25"
+description: "Documentation et exemple pratique d'une boîte grise en gras - CSS provisoire"
+language: "fr"
+altLangPage: "well-bold-en.html"
+tag: "well-bold"
+title: "Boîte grise en gras - CSS provisoire"
+---
+{{>alert-provisional-demos}}
+<span class="wb-prettify all-pre hide"></span>
+
+<dl class="horizontal">
+	<dt>Statut</dt>
+	<dd>Provisiore</dd>
+	<dt>Type</dt>
+	<dd>Conception de Canada.ca</dd>
+	<dt>Dernière revue</dt>
+	<dd>2021-05-25</dd>
+	<dt>Est conforme à</dt>
+	<dd>n/a</dd>
+	<dt>Technical name</dt>
+	<dd><code>well-bold</code></dd>
+</dl>
+
+<h2>Objectif</h2>
+<p>Ceci applique un poid de police de caractère gras à chaque élément de texte à l'intérieur d'un composant boite grise et génère un poid de police de caractère normale lorsque l'élément <code>strong </code> est appliqué.</p>
+
+<h2>Exemple pratique</h2>
+
+<h3>Lien en vedette (Provisiore v1.0)</h3>
+<section class="well well-sm provisional well-bold brdr-0 mrgn-bttm-0">
```

Comment:
Please demo only the well-bold on it's own. Remove the other feature unrelated CSS class